src/main/kotlin/org/jitsi/videobridge/config/SimpleProperty.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.videobridge.config
+
+import org.jitsi.utils.configk.ConfigPropertyAttributes
+import org.jitsi.utils.configk.ConfigResult
+import org.jitsi.utils.configk.FallbackConfigProperty
+import org.jitsi.utils.configk.PropValueRetriever
+import org.jitsi.utils.configk.dsl.legacyConfig
+import org.jitsi.utils.configk.dsl.newConfig
+import kotlin.reflect.KClass
+
+/**
+ * Many configuration properties follow a simple formula:
+ * Check the legacy config file for a property with an 'old' path
+ * Check the new config file for a property with a 'new' path
+ * No conversions of retrieved values are necessary
+ *
+ * This class implements that behavior to make implementing those
+ * property types easier
+ */
+abstract class SimpleProperty<T : Any>(
```

Comment:
I wonder if this should go to jitsi-utils, since we may run into a similar problem once we migrate jibri/jigasi/jicofo. It doesn't import anything from org.jitsi.videobridge